app/common/metrics.py

```diff
@@ -17,6 +20,23 @@
     'serialized_build_time_seconds',
     'Total amount of time that would have been consumed by builds if all work was done serially')
 
+internal_errors = Counter(
+    'internal_errors',
+    'Total number of internal errors',
+    ['type'])
+
+
+class ErrorType(str, Enum):
+    AtomizerFailure = 'AtomizerFailure'
+    NetworkRequestFailure = 'NetworkRequestFailure'
+    PostBuildFailure = 'PostBuildFailure'
+    SetupBuildFailure = 'SetupBuildFailure'
+    SubjobWriteFailure = 'SubjobWriteFailure'
+    ZipFileCreationFailure = 'ZipFileCreationFailure'
+
+    def __str__(self):
+        return self.value
```

Comment:
Yeah, I figured that was it. `prometheus_client` must be using `str()` instead of `format()` internally. The reason the other enums inherit from `str` is because `json.dumps` must be using `format()` internally so we get the correct formatting for free.